demo-projects/meetup/site/pages/event.js

```diff
@@ -62,7 +62,7 @@ export default class Event extends Component {
                 title={name}
               >
                 <p css={{ fontWeight: 100 }}>{locationAddress}</p>
-                <Html markup={description} />
+                {description ? <Html markup={description.replace(/style=".*?"/g, '')} /> : null}
```

Comment:
Same as comment above, if this logic is required it should be encapsulated somewhere (like a `Description` component if not the `Html` component itself) rather than being repeated through the codebase